test/controllers/user.test.ts

```diff
@@ -280,5 +291,22 @@ describe("User controller tests", () => {
 
             expect(user).to.deep.equal(expectedBody);
         });
+        
+        it("status 200: returns successfully deleted user by its userId", async () => {
+            const newTestUserData: IUser = {
+                email: "instance@gmail.com",
+                githubToken: "newToken",
+                githubUsername: "newExample",
+                githubRepo: "https://github.com/CompeteMcgill/challenge-template2",
+                scores: []
+            };   
+            const newTestUser = await userDBInteractions.create(newTestUserData);
+
+            const usersBeforeDelete = await userDBInteractions.all();
+            expect(usersBeforeDelete.length).to.equal(2);
```

Comment:
No need to create new user for testing. You can delete the pre-created "testUser", use userDBInteractions to find it (shouldn't be able to find it), then expect the user you tried to find to be undefined/null:

            const { body: user } = await chai.request(app).delete("/users/" + testUser._id);
            const foundUser = await userDBInteractions.find(testUser._id);

            expect(foundUser).to.be.null;